### PR TITLE
v4.4 Changelog

### PR DESCRIPTION
**Pull request by esethna**
Opening v4.4 Changelog and starting "v4.4-documentation" branch

### CODE REVIEW

**Comment by jasonblais** [comment]
Ready for editor and PM review. Two outstanding items:
 - @esethna please help with elasticsearch config settings
 - Some API changes haven't been doced, have pinged the respective devs to help.

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,220 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+**Added routes (API v4)**
+- `POST` at `/users/token/enable`
+  - Re-enables a personal access token that was previously disabled.
+- `POST` at `/users/token/disable`
+  - Disables a personal access token and deletes any sessions that uses the token. The token can be re-enabled using `/users/tokens/enable.`
+- `POST` at `/users/{user_id}/sessions/revoke/all`
+  - Revokes all user sessions from the provided user id and session id strings.
+
+- Plugin APIs // XXX Pinged JW to help
```
Comment (esethna) [inline source/administration/changelog.md]: Still needs help from @jwilander

**Comment by esethna** [comment]
Added missing elasticsearch info and made a few tweaks. Changelog looks good otherwise.

File: source/administration/changelog.md (hunk 1 above)
Comment (jwilander) [reply source/administration/changelog.md]:
```
- `POST` at `/plugins`
  - Upload a plugin in a compressed .tar.gz
- `GET` at `/plugins`
  - Get a list of active and inactive plugins
- `DELETE` at `/plugins/{plugin_id}`
  - Remove a previously uploaded plugin
- `POST` at `/plugins/{plugin_id}/activate`
  - Activate an installed plugin
- `POST` at `/plugins/{plugin_id}/deactivate`
  - Deactivate an active plugin
- `GET` at `/plugins/webapp`
  - Get a list of plugin manifests for active plugins with webapp components
```

**Pull request merged**